plugins/module_utils/kibana.py

```diff
@@ -215,7 +215,7 @@ def get_all_agent_policys(self):
     agent_policy_objects = self.send_api_request(endpoint, 'GET')
     return agent_policy_objects
 
-  def create_agent_policy(self, agent_policy_id, agent_policy_name, agent_policy_desc):
+  def create_agent_policy(self, agent_policy_id, agent_policy_name, agent_policy_desc, namespace="default"):
```

Comment:
Since namespace is set to default to "default" in both modules, I think it's better to drop the default argument here.

Reply:
I think we're splitting hairs at this point. We tested and it works right?

Reply:
I'm making suggestions based on issues that have bit me in the past.

Reply:
No worries, we're just trying to get it pushed in a working state. We can circle back and clean up if we need to.